agent/runner/actions/mysql_explain_action.go

```diff
@@ -35,6 +36,7 @@ import (
 const (
 	errNoDatabaseSelectedCode    = "Error 1046 (3D000)"
 	errNoDatabaseSelectedMessage = "Database name is not included in this query. Explain could not be triggered without this info"
+	errNoDatabaseProvidedMessage = "Captured database name is empty and it is not even included in query. Run explain is not possible"
```

Comment:
TBH I don't really understand what is the difference between these two errors

Reply:
We need two different error messages for:
1. Case schema is captured by MySQL and not included in query (explain should be possible), but explain failed anyway for some reason. Could be bug.
2. Case schema is not captured by MySQL and not included in query then explain is not possible at all. Not bug.

Without two different messages its very hard to figure out what happened from customers reports and then we need ask them to debug problem to decide which situation happened.